src/app/reset-password/reset-password-form/ResetPasswordForm.tsx

```diff
@@ -0,0 +1,53 @@
+'use client';
+import { useState } from 'react';
+import * as Label from '@radix-ui/react-label';
+import { Button, Flex } from '@radix-ui/themes';
+
+import styles from './resetPassword.module.css';
+import { ResetPasswordTypes } from './types';
+
+const ResetPasswordForm = ({ resetPassword }: ResetPasswordTypes) => {
+  const [email, setEmail] = useState<string>('');
+  const [error, setError] = useState<string>('');
+  const [message, setMessage] = useState<string>('');
+
+  const onResetPasswordHandler = async () => {
+    setError('');
+    if (email.includes('@')) {
+      const result = await resetPassword(email);
+      if (result.error) {
+        setError(result.message);
+      } else {
+        setMessage(result.message);
+      }
+    } else {
+      setError('Please enter a valid email');
+    }
+  };
```

Comment:
a good pattern to avoid nesting if statement - do negative return as soon as posible, for example
`const onResetPasswordHandler = async () => {
  setError('');

  if (!email.includes('@')) {
    setError('Please enter a valid email');
    return;
  }
  const result = await resetPassword(email);

  if (result.error) {
      setError(result.message);
  } else {
      setMessage(result.message);
  }
};`

Reply:
oh yes, it seems shorter. thanks